llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -1806,7 +1806,7 @@ OpenMPIRBuilder::createTask(const LocationDescription &Loc,
         ++P;
       }
 
-      DepArrayPtr = Builder.CreateBitCast(DepArray, Builder.getInt8PtrTy());
+      DepArrayPtr = Builder.CreateBitCast(DepArray, Builder.getPtrTy());
```

Comment:
Can remove this call entirely.

Reply:
`DepArrayPtr = DepArrays; ` would still be needed afaict (so not just remove)

Reply:
Yeah. I'm going to rename DepArrayPtr and change the DepArray declaration in this if to an assignment to DepArray declared before the.